README.md

```diff
@@ -0,0 +1,115 @@
+# claiming-app-data
+Vesting data converter and proofs generator.
+Guardian data converter and image loader. Images are scaled and converted to PNG.
+
+## Guardians
+
+Place csv with guardian data under `guardians/assets/guardians.csv`.
+Not all guardian images could be cropped to square dimensions without distorting the image. We have created square image versions for guardians that do not have square images and placed them under
+`guardians/assets/images`. This image collection should be extended if another guardian with not square image is added. These images will be taken for a guardian during conversion instead of initially provided image.
+
+Naming convention for image files under `guardians/assets/images`:
+```
+<guardian name>.png
+```
+
+### Generating
+TBD
+
+Generated data will be placed under `../data/guardians`
+
+Guardians list: `../data/guardians/guardians.json`
+
+Guardian images: `../data/guardians/images/`
+
+### Deploying to staging
+
+1. Add generated files from data/ folder to repository and commit them.
```

Comment:
I was under the impression that this would happen automatically in a github action

Reply:
Generating proofs for all allocations might take several hours. This is the main reason for doing this locally and uploading results to github.